packages/picasso-provider/src/Picasso/utils/spacings.ts

```diff
@@ -0,0 +1,18 @@
+import type { SizeType, SpacingType } from '../config/spacings'
+import { SpacingEnum, PicassoSpacing } from '../config/spacings'
+
+const isNumericSpacing = (spacing?: SpacingType): spacing is number => {
+  if (!spacing) {
```

Comment:
This will fail when spacing is `0`, which is not what we want for this case, we still need to provide a margin/padding of `0` in that case, here we need to specifically test for nullity:
```suggestion
  if (spacing != null) {
```